### PR TITLE
Re-render to CRAN versions

### PR DESCRIPTION
**Pull request by EmilHvitfeldt**
WIP

### CODE REVIEW

File: docs/06_ml_regression.md [hunk 1]
```diff
@@ -1401,7 +1412,9 @@ final_grid
 #> 6       6000
 ```
 
-Now it's time for tuning. Instead of using `fit_resamples()` as we have throughout this chapter, we are going to use `tune_grid()`, a function that has a very similar set of arguments. We pass this function our workflow (which holds our preprocessing recipe and SVM model), our resampling folds, and also the grid of possible parameter values to try. Let's save the predictions so we can explore them in more detail, and let's also set custom metrics instead of using the defaults. Let's compute RMSE, mean absolute error, and mean absolute percent error during tuning.
```
[the remark this answers is not in the document]
Comment (EmilHvitfeldt) [reply docs/06_ml_regression.md:1404]: Yes, this paragraph is still important, I'll add it back in

File: docs/02_tokenization_files/figure-html/ngramtokens-1.svg [hunk 2]
```diff
@@ -39,630 +39,630 @@
 <polyline points='158.76,279.72 158.76,39.07 ' style='stroke-width: 0.53; stroke: #DEDEDE; stroke-linecap: butt;' />
 <polyline points='246.65,279.72 246.65,39.07 ' style='stroke-width: 0.53; stroke: #DEDEDE; stroke-linecap: butt;' />
 <polyline points='334.55,279.72 334.55,39.07 ' style='stroke-width: 0.53; stroke: #DEDEDE; stroke-linecap: butt;' />
-<circle cx='81.95' cy='77.11' r='1.95' style='stroke-width: 0.71; stroke: #6EC95D; stroke-opacity: 0.80; fill: #6EC95D; fill-opacity: 0.80;' />
```
Comment (juliasilge) [inline docs/02_tokenization_files/figure-html/ngramtokens-1.svg:42]: These diffs are due to the jitter; we might want to [set a seed](https://stackoverflow.com/a/62692568/5468471).

File: docs/02_tokenization_files/figure-html/tokendists-1.svg [hunk 3]
```diff
@@ -25,20 +25,20 @@
   </clipPath>
 </defs>
 <g clip-path='url(#cpMTI5LjI1fDQ5OC41MnwzNy41NHwyNzkuNzI=)'>
-<polyline points='205.35,279.72 205.35,37.54 ' style='stroke-width: 0.53; stroke: #EBEBEB; stroke-linecap: butt;' />
+<polyline points='205.34,279.72 205.34,37.54 ' style='stroke-width: 0.53; stroke: #EBEBEB; stroke-linecap: butt;' />
```
Comment (juliasilge) [inline docs/02_tokenization_files/figure-html/tokendists-1.svg:28]: Same as above: switch to using a seed for jitter

File: docs/06_ml_regression.md [hunk 4]
```diff
@@ -862,7 +867,13 @@ Lemmatization is, like choices around n-grams and stop words, part of data prepr
 
 ```r
 spacyr::spacy_initialize(entity = FALSE)
+```
+
+```
+#> NULL
+```
```
Comment (juliasilge) [inline docs/06_ml_regression.md:874]: I remember we had eventually gotten rid of this but now IT IS BACK 🥴

Not the end of the world, but I wonder what changed?

File: docs/10_dl_cnn.md [hunk 5]
```diff
@@ -935,8 +935,8 @@ plot_text_explanations(explanation)
...
+<div id="htmlwidget-f547c97c3d4a26a67991" style="width:100%;height:auto;" class="plot_text_explanations html-widget"></div>
+<script type="application/json" data-for="htmlwidget-f547c97c3d4a26a67991">{"x":{"html":"<div style=\"overflow-y:scroll;font-family:sans-serif;height:100%\"> <p> <span class='negative_1'>Fun<\/span> <span class='positive_1'>and<\/span> <span class='positive_1'>exciting<\/span> <span class='positive_4'>dice<\/span> <span class='positive_1'>game<\/span> <span class='positive_1'>for<\/span> <span class='positive_1'>the<\/span> <span class='positive_1'>whole<\/span> <span class='negative_1'>family<\/span> <\/br> <sub>Label predicted: 2 (99.94%)<br/>Explainer fit: 0.76<\/sub> <\/p><br/><p> <span class='negative_1'>Fun<\/span> <span class='positive_1'>and<\/span> <span class='positive_1'>exciting<\/span> <span class='positive_4'>dice<\/span> <span class='positive_1'>game<\/span> <span class='positive_1'>for<\/span> <span class='positive_1'>the<\/span> <span class='negative_1'>family<\/span> <\/br> <sub>Label predicted: 2 (99.91%)<br/>Explainer fit: 0.68<\/sub> <\/p> <\/div>"},"evals":[],"jsHooks":[]}</script>
```
Comment (juliasilge) [inline docs/10_dl_cnn.md:939]: Did you look at this, plus the next bit? Does it still make sense with the text?

File: docs/10_dl_cnn_files/figure-html/limeplotfeatures-1.svg [hunk 6]
```diff
@@ -25,25 +25,24 @@
...
-<rect x='158.18' y='194.57' width='9.99' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #5BBCAC; fill-opacity: 0.90;' />
-<rect x='158.18' y='109.38' width='15.88' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #5BBCAC; fill-opacity: 0.90;' />
-<rect x='158.18' y='137.78' width='13.33' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #5BBCAC; fill-opacity: 0.90;' />
-<rect x='158.18' y='95.18' width='19.29' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #5BBCAC; fill-opacity: 0.90;' />
-<rect x='147.82' y='180.38' width='10.36' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #D5D587; fill-opacity: 0.90;' />
+<polyline points='96.85,223.68 96.85,64.65 ' style='stroke-width: 0.53; stroke: #EBEBEB; stroke-linecap: butt;' />
```
Comment (juliasilge) [inline docs/10_dl_cnn_files/figure-html/limeplotfeatures-1.svg:28]: This has changed quite a lot, but I think it is OK with the text. Maybe you can confirm?

Also, I have noticed that in the current online version [we show this same plot twice](https://smltar.com/dlcnn.html#lime) but with different colors; we should fix that.

File: docs/10_dl_cnn_files/figure-html/unnamed-chunk-40-1.svg [hunk 7]
```diff
@@ -25,25 +25,24 @@
...
-<rect x='158.18' y='194.57' width='9.99' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #4682B4; fill-opacity: 0.90;' />
-<rect x='158.18' y='109.38' width='15.88' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #4682B4; fill-opacity: 0.90;' />
-<rect x='158.18' y='137.78' width='13.33' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #4682B4; fill-opacity: 0.90;' />
-<rect x='158.18' y='95.18' width='19.29' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #4682B4; fill-opacity: 0.90;' />
-<rect x='147.82' y='180.38' width='10.36' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #B22222; fill-opacity: 0.90;' />
+<polyline points='96.85,223.68 96.85,64.65 ' style='stroke-width: 0.53; stroke: #EBEBEB; stroke-linecap: butt;' />
```
Comment (juliasilge) [inline docs/10_dl_cnn_files/figure-html/unnamed-chunk-40-1.svg:28]: This is the first, wrong-colored version that we do not want to include.

File: docs/02_tokenization_files/figure-html/ngramtokens-1.svg (hunk 2 above)
Comment (EmilHvitfeldt) [reply docs/02_tokenization_files/figure-html/ngramtokens-1.svg:42]: Done!

File: docs/02_tokenization_files/figure-html/tokendists-1.svg (hunk 3 above)
Comment (EmilHvitfeldt) [reply docs/02_tokenization_files/figure-html/tokendists-1.svg]: Done!

File: docs/06_ml_regression.md (hunk 4 above)
Comment (EmilHvitfeldt) [reply docs/06_ml_regression.md:874]: I don't see. what changed. What we did in other places was to have `echo=FALSE` in the chunk. But this chunk has something we want to be printed. I'm gonna pull out the spacy call and `result='hide'` it

File: docs/10_dl_cnn_files/figure-html/unnamed-chunk-40-1.svg (hunk 7 above)
Comment (EmilHvitfeldt) [reply docs/10_dl_cnn_files/figure-html/unnamed-chunk-40-1.svg:28]: Deleted!

File: docs/10_dl_cnn_files/figure-html/limeplotfeatures-1.svg (hunk 6 above)
Comment (EmilHvitfeldt) [reply docs/10_dl_cnn_files/figure-html/limeplotfeatures-1.svg]: This is fixed now

File: docs/10_dl_cnn.md (hunk 5 above)
Comment (EmilHvitfeldt) [reply docs/10_dl_cnn.md]: This makes sense in the newest commit

**Review by juliasilge — APPROVED** [review]
I believe this is ready to go 👍

**Pull request merged**